source/dem/particle_particle_contact_force.cc

```diff
@@ -187,7 +109,7 @@ ParticleParticleContactForce<dim, contact_model, rolling_friction_model>::
                        particle_two_properties[PropertiesIndex::dp]) -
                 particle_one_location.distance(particle_two_location);
 
-              if (normal_overlap > 0.0)
+              if (normal_overlap > force_calculation_threshold_distance)
```

Comment:
The threshold distance should be calculated locally before we enter the loop instead of just setting it as a member variable. It's not a wise idea to set it as a member variable.

Reply:
this goes will my last reply.

Reply:
it's now being computed at the start of the function.